apps/src/templates/rubrics/rubrics.module.scss

```diff
@@ -135,7 +175,6 @@ p.aiTokenText {
   margin-bottom: 0;
 }
 
-
```

Comment:
not sure where all these deleted lines came from but they look like a positive change